readme.markdown

```diff
@@ -786,6 +786,23 @@ To summarise, the steps for release are:
 Once the new tag is in the remote repository, Nimble will be able to detect
 the new version.
 
+##### Git Version Tagging
```

Comment:
I don't think this information is important enough to include in the readme. We should be making the Nimble docs as streamlined as possible.

Reply:
There is an implicit relationship between the git tag label and the version info that nimble displays and uses.  It's valuable to let users know some info about it so they don't have to guess or dive into the code just to remember how they should be tagging their releases.  I'm still going to circle back to this and see if I can streamline the docs a bit more.

Reply:
I would prefer if you instead emphasise that this is how users should be tagging in the above section. All the details about version sorting is unnecessary.